app/models/feedback.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Feedback < ApplicationRecord
```

Comment:
As I commented in #3: 

> That was just the first thing that came to my mind, but I like issues. For replies, I think I’d prefer something more generic like reply. I directly associated feedback with a certain type of reply, but maybe that’s just me messing with the meaning of English words… ;)

What do you think?

Reply:
Issues + replies? I'm OK with that

Reply:
Requests + Replies now